fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -292,7 +293,7 @@ class ThresholdOptimizer(ClassifierMixin, BaseEstimator):
 
     :param estimator: An untrained estimator that will be trained, and
         subsequently its output will be postprocessed
-    :type estimator: An untrained estimator
+    :type estimator: An untrained estimator, default=SGDClassifier(loss='log')
```

Comment:
Do we want to provide a default? If so, why that one? Just generally wondering, but I don't have a strong opinion

Reply:
I thought we couldn't, since this was mutually exclusive with `unconstrained_predictor` ?

Reply:
Hmmmm…. would be related to the `warm_start` below.